Libraries/TestHelpers.psm1

```diff
@@ -78,7 +78,7 @@ function Create-TestResultObject()
 }
 
 # Upload a single file
-function Upload-RemoteFile($uploadTo, $port, $file, $username, $password, $usePrivateKey, $maxRetry) {
+function Upload-RemoteFile($uploadTo, $port, $file, $username, $password, $usePrivateKey, $pscpTimeout, $maxRetry) {
```

Comment:
can you set a default value here as well?

Reply:
done